renderers/photoaudiohaptics.schema.json

```
@@ -34,10 +34,10 @@
         }
     },
     "properties": {
-        "image": {
+        "graphic": {
             "type": "string",
-            "description": "A base64 encoding of the image as a string.",
-            "pattern": "data:image\/[^;]+;base64[^\"]+"
+            "description": "A base64 encoding of the graphic as a string.",
+            "pattern": "data:graphic\/[^;]+;base64[^\"]+"
```

This is a [media type](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types).
```suggestion
            "pattern": "data:image\/[^;]+;base64[^\"]+"
```